include/roboteam_world/filters/WorldFilter.h

```diff
@@ -0,0 +1,39 @@
+#ifndef ROBOTEAM_WORLD_KALMANFILTER_H
+#define ROBOTEAM_WORLD_KALMANFILTER_H
+
+#include <filters/RobotFilter.h>
+#include <filters/BallFilter.h>
+#include "roboteam_proto/WorldRobot.pb.h"
+#include "roboteam_proto/World.pb.h"
+
+namespace world {
+
+    /**
+     * @author Rolf van der Hulst
+     * @date November 2019
+     * @brief class that tracks a world. The function of this class is to determine when to create and delete filters,
+     * and to pass the relevant incoming information to the relevant filter(s)
+     */
+    class WorldFilter {
+    public:
+        WorldFilter();
+        void addFrame(const proto::SSL_DetectionFrame &msg);
+        proto::World getWorld(double time);
+    private:
+        void update(double time, bool extrapolateLastStep);
+        typedef std::map<int, std::vector<std::unique_ptr<RobotFilter>>> robotMap;
```

Comment:
A robotID can point to multiple RobotFilters. Does this assume that robots with the same ID can be on the field?

Reply:
Yes.